chatkit-core/src/main/kotlin/com/pusher/chatkit/model/mappers/Room.kt

```diff
@@ -0,0 +1,35 @@
+package com.pusher.chatkit.model.mappers
+
+import com.pusher.chatkit.model.network.ReadStateApiType
+import com.pusher.chatkit.model.network.RoomApiType
+import com.pusher.chatkit.model.network.RoomMembershipApiType
+import com.pusher.chatkit.rooms.Room
+
+internal fun mapToRoom(
```

Comment:
Better packaging: `com.pusher.chatkit.rooms.api.JoinedRoomApiMapper`. A class so that it can be created once (in the future by DI) and more easily swapped if needed in the future. Also having a class conveys more needed information about what are mapping from and makes it easier / more natural to navigate to (ctrl+n).